CodeEdit/Features/AppPreferences/Model/Text Editing/TextEditingPreferences.swift

```diff
@@ -8,6 +8,31 @@
 import AppKit
 import Foundation
 
+extension String.Encoding: Codable {
+    static var allAscii: [String.Encoding] = [String.Encoding.ascii, String.Encoding.nonLossyASCII]
+    static var allJapanese: [String.Encoding] = [String.Encoding.iso2022JP, String.Encoding.japaneseEUC]
+    static var allLatin: [String.Encoding] = [String.Encoding.isoLatin1,
+                                              String.Encoding.isoLatin2,
+                                              String.Encoding.macOSRoman
+                                            ]
```

Comment:
Please mind the [Code Style](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#parameter-alignment)

```suggestion
    static var allLatin: [String.Encoding] = [
        .isoLatin1,
        .isoLatin2,
        .macOSRoman
    ]
```